### ISSUE TITLE
Globals aren't highlighted

### ISSUE THREAD

**Issue by swrobel** [tree-sitter/tree-sitter-ruby#104]
They were in the old (non-tree-sitter) grammar, but they don't appear to be anymore:
<img width="116" alt="screen shot 2019-02-26 at 6 06 23 pm" src="https://user-images.githubusercontent.com/157270/53460230-44760e80-39f1-11e9-84d9-0594feedd409.png">

**Comment by tclem** [comment]
Based on the following testing, I believe the issue going to need to be fixed over in the [atom package](https://github.com/atom/language-ruby). Tree-sitter-ruby appears to be doing the right thing:

``` bash
❯ cat test.rb
@instance_var
@@class_var
$global_var
❯ tree-sitter parse test.rb
(program [0, 0] - [3, 0]
  (instance_variable [0, 0] - [0, 13])
  (class_variable [1, 0] - [1, 11])
  (global_variable [2, 0] - [2, 11]))
```

Do you mind if I transfer this issue over there?

**Comment by swrobel** [comment]
This seems fixed in Atom 1.37 so I'm closing out.
